core/storage/suggestion/gae_models.py

```diff
@@ -2004,9 +2025,9 @@ def export_data(
 
 class TranslationSubmitterTotalContributionStatsModel(base_models.BaseModel):
     """Records the Total Translation contribution stats and data of
-    recent_review keyed per (contributor_id, language_code) tuple.
+    recent_review keyed per (language_code, contributor_id) tuple.
     Its IDs will be in the following
-        structure: [contributor_id][language_code]
+        structure: [language_code].[contributor_id]
```

Comment:
Nit: Why the weird line break and indentation? (You can put "structure:" on the previous line and then have the rest on the next line.)

Reply:
Actually we were exceeding line lenght therefore I did line break before structure.